tests/projects/Sample_NETCoreSDK_FSharp_Library_netstandard1.6/Sample_NETCoreSDK_FSharp_Library_netstandard1.6.fsproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <DebugType>portable</DebugType>
+    <AssemblyName>Sample_NETCoreSDK_FSharp_Library_netstandard1.6</AssemblyName>
+    <PackageId>Sample_NETCoreSDK_FSharp_Library_netstandard1.6</PackageId>
```

Comment:
`AssemblyName` and `PackageId` are defaulted to `.fsproj` filename, so these are not needed
`<DebugType>portable</DebugType>` is implicit default, remove it
`NetStandardImplicitPackageVersion` is not needed